meta_review/handler.py

```diff
@@ -0,0 +1,412 @@
+from dateutil import parser
+import json
+import logging
+import os
+
+from django.utils import timezone
+
+from meta_review.dump_to_db import dump_all
+from meta_review.load_from_db import load_all
+from meta_review.models import Participant
+
+
+def parse_time(time):
+    """
+    parse string to datetime
+    :param time: a string represents time, e.g. 2018-05-09T11:19:26Z
+    :return: an offset-aware datetime object
+    """
+    if time is None:
+        return None
+
+    return parser.parse(time)
+
+
+class MetaReviewHandler:
+    """
+    This is the class responsible for scraping provided information (reviews,
+    reactions), processing them and dumping into Django database
+    """
+
+    def __init__(self, content, date):
+        """
+        Constructs a new ``MetaReviewHandler``
```

Comment:
run pydocstyle on your code.  It will tell you to add a period at the end here.

Ideally get the relevant bear working on this repo when you can.